src/solvers/solvers.jl

```diff
@@ -0,0 +1,3 @@
+include("dg/dg.jl")
+include("dg_common.jl")
+include("dg_structured/dg.jl")
```

Comment:
```suggestion
include("dg_structured/dg.jl")
include("dg_common.jl")
```
Otherwise it would trigger me to wonder if there's anything in the structured DG solver that is not in the tree DG solver that depends on stuff in `dg_common.jl`